### PR TITLE
Added version file, updated README

### PR DESCRIPTION
**Pull request by yathi-sysdig**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,14 +118,5 @@ Here are examples of using the inline scanner in different pipelines.
...
-    
-    Docker client access to pull images from Dockerhub
-    
-    Internet Access to post results to Sysdig Secure
-    
-#### Scan Result PDF when running the script with -R option
-    ![Screenshot](https://user-images.githubusercontent.com/39659445/75296350-c6a23a00-57e1-11ea-9a55-d1d0b8b7ac1d.png "Scan result PDF")    
\ No newline at end of file
+#### Produce a PDF report of the scan results using the -R option
+<img width="1589" alt="scan-results pjg" src="https://user-images.githubusercontent.com/48695865/76269075-8dc88300-622d-11ea-9537-79b3c70596be.png">
```
Comment (haresh-suresh) [inline README.md]: Can we use these screenshots instead ? Little more verbose ?
```
<img width="1377" alt="node-scan-result-pg1" src="https://user-images.githubusercontent.com/39659445/76037687-8dae4780-5efc-11ea-9f26-9347a5c4334c.png">
  <img width="1377" alt="node-scan-result-pg2" src="https://user-images.githubusercontent.com/39659445/76037691-90a93800-5efc-11ea-8fe1-ebfe8a13110d.png">
```

File: inline_scan.sh [hunk 2]
```diff
@@ -435,7 +435,7 @@ print_scan_result_summary_message() {
         fi
         ENCODED_TAG=$(urlencode ${FULLTAG})
         echo "View the full result @ ${SYSDIG_BASE_SCANNING_URL}/#/scanning/scan-results/${ENCODED_TAG}/${SYSDIG_IMAGE_DIGEST}/summaries"
-        printf "You can also run the script with -R option for more info.\n"
+        printf "A PDF report of the scan results can be generated with -R option.\n"
```
Comment (haresh-suresh) [inline inline_scan.sh]: I think the sample output / options in both CI and normal README need to be updated accordingly

Comment (yathi-sysdig) [reply inline_scan.sh]: yup

**Review by haresh-suresh — APPROVED** [review]
🚢

**Pull request merged**